queues.go

```diff
@@ -1,68 +1,109 @@
 package control
 
+// Region is an enum of the possible queue regions.
 type Region string
 
+// UsEast1A is the us east 1 a region.
 const UsEast1A Region = "us-east-1-a"
+
+// EuWest1A is the eu west 1 a region.
```

Comment:
Tbh I was thinking that we can commit this comment (and the one above) as code is pretty straightforward. Maybe add context on why const is needed instead? As far as I could see also this const is used in test only - maybe move it there?

Reply:
I assume you mean omit? It's best to have a comment on everything, no matter how trivial as these are doc comments and are used for the package documentation page.

UsEast1A is a const of type Region, not string. We give region it's own type so that when a user encounters a region they see it as on opaque type with a bunch of consts as the possible values. The type however is backed by a string because that's what it is on the json level.

This type is used in our public API but we don't actually call our own API except for testing. This is why instances of it only appear in tests.

Reply:
Sorry, this was silly on my side - all makes sense, thanks for explaining!